services/build-status.js

```diff
@@ -27,6 +27,7 @@ const otherStatuses = [
   'cancelled',
   'expired',
   'initiated',
+  'no builds',
```

Comment:
We should probably update the corresponding test in [build-status.spec.js](https://github.com/badges/shields/blob/master/services/build-status.spec.js#L67) with this entry since that test includes the other status